diff --git a/dist/vitest-plugin/index.js b/dist/vitest-plugin/index.js
index 720ba80f7b91893af3c6a6830d0d8bcd5c474106..e66ce3120570812ef540c37110b0ed917c5849b4 100644
--- a/dist/vitest-plugin/index.js
+++ b/dist/vitest-plugin/index.js
@@ -2603,7 +2603,7 @@ var WORKING_DIR = process.cwd(), defaultOptions = {
       let includeStories = stories.map((story) => {
         let storyPath;
         return typeof story == "string" ? storyPath = story : storyPath = `${story.directory}/${story.files ?? DEFAULT_FILES_PATTERN}`, join(finalOptions.configDir, storyPath);
-      }).map((story) => relative(finalOptions.vitestRoot, story));
+      }).map((story) => relative(finalOptions.vitestRoot, story).replaceAll(sep, "/"));
       finalOptions.includeStories = includeStories;
       let projectId = oneWayHash(finalOptions.configDir), internalSetupFiles = [
         "@storybook/addon-vitest/internal/setup-file",
@@ -2748,7 +2748,7 @@ var WORKING_DIR = process.cwd(), defaultOptions = {
           }
     },
     async transform(code, id) {
-      let relativeId = relative(finalOptions.vitestRoot, id);
+      let relativeId = relative(finalOptions.vitestRoot, id).replaceAll(sep, "/");
       if ((0, import_micromatch.match)([relativeId], finalOptions.includeStories).length > 0)
         return vitestTransform({
           code,
diff --git a/dist/vitest-plugin/test-utils.js b/dist/vitest-plugin/test-utils.js
index 9048740b2c8fba5a813a5541180dbd8bc2ec5984..4b34760f3cb19943ef4691ade3fb78973f2a2c8b 100644
--- a/dist/vitest-plugin/test-utils.js
+++ b/dist/vitest-plugin/test-utils.js
@@ -72,7 +72,7 @@ var DEFAULT_VIEWPORT_DIMENSIONS = {
 };
 
 // src/vitest-plugin/test-utils.ts
-var convertToFilePath = (url) => url.replace(/^file:\/\//, "").replace(/^\/+([a-zA-Z]:)/, "$1").replace(/%20/g, " "), testStory = ({
+var convertToFilePath = (url) => decodeURI(url).replace(/^file:\/\//, "").replace(/^\/+([a-zA-Z]:)/, "$1"), testStory = ({
   exportName,
   story,
   meta,
